### DISCUSSION TITLE
Installer now successful. Clarification re webcam 8470p?

### DISCUSSION THREAD

**Discussion by DANW-NZ** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#131] [Q&A] [answered]
I used a different USB drive on a different mac and managed to build the OC USB installer just fine. 

Fairly smooth sailing installing Monterey too, everything seems to work, sound, WIFI, DP output, a totally awesome tool. 

One final question though… and forgive me if it has been answered, but Chris, can you clarify what needs to be done to use a webcam? I understand SIP needs to be disabled for Monterey to boot and this stops Monterey from accessing the webcam. Is there anyway to have a functional Webcam in Monterey? If not, what exactly do I need to do in Big Sur to have a functional webcam? I’ve had a bit of experience modifying config.plist for OC installs on other machines I own, but not for Monterey. I need to do a bit of intermittent Zoom conferencing with the machine so unfortunately do need it… 
 
Thanks in advance for taking the time to answer.

**Comment by chris1111** [comment]
CAM open with FaceTime ?

Try this, open config.plist with OpenCore configurator, dont update it!
Change the value here
`EF0F0000 `

![Screen Shot 2022-06-21 at 3 14 04 PM](https://user-images.githubusercontent.com/6248794/174880796-e12f7ce2-1352-49fc-916c-4ef779970be3.png)

To this
`00080000`
![Screen Shot 2022-06-21 at 3 15 10 PM](https://user-images.githubusercontent.com/6248794/174880891-994e62f7-a163-4532-921c-db296f9afd40.png)

Reboot the laptop then Reset NVRAM OpenCore 
click Key TAB then enter to Restart
let me know if that work

**Comment by DANW-NZ** [comment]
No camera didn’t work with FaceTime.
I’ll try your suggestion, and thanks so much for the advice and all the work- it’s totally awesome.
I’ll drop you some $$ via PayPal once I get it all working.

Daniel


Sent from my iPhone

On 22/06/2022, at 07:23, chris1111 ***@***.***> wrote:

﻿

CAM open with FaceTime ?

Try this, open config.plist with OpenCore configurator, dont update it!
Change the value here
EF0F0000

[Screen Shot 2022-06-21 at 3 14 04 PM]<https://github.com/assets/6248794/e12f7ce2-1352-49fc-916c-4ef779970be3>

To this
00080000
[Screen Shot 2022-06-21 at 3 15 10 PM]<https://github.com/assets/6248794/994e62f7-a163-4532-921c-db296f9afd40>

Reboot the laptop then Reset NVRAM OpenCore
click Key TAB then enter to Restart
let me know if that work

**Comment by chris1111** [comment]
see Camera on About this Mac / CAM and / USB
Post the 2 picture here

**Comment by DANW-NZ** [comment]
Sure thing, I’m at work now and won’t get to it until tomorrow, but will post then. It definitely did come up as detected in the system report and the led next to the camera comes on when FaceTime is activated, just no image.


Sent from my iPhone

On 22/06/2022, at 10:06, chris1111 ***@***.***> wrote:

﻿

see Camera on About this Mac / CAM and / USB
Post the 2 picture here

**Comment by DANW-NZ** [comment]
![Screen Shot 2022-06-22 at 2 39 31 PM](https://user-images.githubusercontent.com/107822196/175144215-0f24c534-6a54-47dc-bd87-79aa938c68b2.png)
![Screen Shot 2022-06-22 at 2 39 19 PM](https://user-images.githubusercontent.com/107822196/175144234-60eca77c-cf9a-4429-8028-c54c99466c26.png)

I altered the CSR value as suggested but no change. I plugged in an external USB camera and that worked fine though.
Also tried with Zoom and no go. 

I haven't actually run windows on this laptop... I'll install that next just to make sure the webcam is actually functional, it would be kind of embarrassing if it's faulty! I'm pretty sure it does work, the LED next to it activates when FaceTime tries to use it and clearly it's detected by Monterey. 

Any more suggestions?

**Comment by chris1111** [comment, marked as answer]
I have the same Camera all is work here
FaceTime
You need to log to the Apple ID
![Screen Shot 2022-06-22 at 6 36 24 PM](https://user-images.githubusercontent.com/6248794/175166487-8343b2e5-d225-4863-a471-52eb91c54eb0.png)

**Comment by DANW-NZ** [comment]
Are you using Monterey?

Sent from my iPhone

On 23/06/2022, at 10:42, chris1111 ***@***.***> wrote:

﻿

I have the same Camera all is work here
FaceTime
You need to log to the Apple ID
[Screen Shot 2022-06-22 at 6 36 24 PM]<https://user-images.githubusercontent.com/6248794/175166244-d03203b0-11a6-4890-9f59-318f4fc1f1a4.png>

**Comment by chris1111** [comment]
yes 12.4 21F79

**Comment by DANW-NZ** [comment]
Camera works fine in windows… a mystery
Do I need to generate my own serial number etc for your install tool? It looks like one has been generated automatically, I wondered if that had something to do with it.


Sent from my iPhone

On 23/06/2022, at 11:06, chris1111 ***@***.***> wrote:

﻿

yes 12.4 21F79

**Comment by chris1111** [comment]
no impact

**Comment by DANW-NZ** [comment]
Would napping the internal webcam to a different USB port make a difference?

Sent from my iPhone

On 23/06/2022, at 13:46, chris1111 ***@***.***> wrote:

﻿

no impact
